external/board/radxa03w/rootfs_overlay/etc/init.d/network.sh

```diff
@@ -0,0 +1,14 @@
+DELAY=1
+
+## TODO: REMOVE
```

Comment:
So is this needed or not?

Additionally, is there any reason why this isn't directly in S40network?

Reply:
Not needed anymore.
It wasn't in S40network to allow boot to continue while netinit runs in the background